### PR TITLE
Minor restructure and refactoring

### PR DESCRIPTION
**Pull request by D-T-666**
Moved some functions from **`public/sketch.js`** to **`public/script.js`**, only left dom related and p5js related function in **`public/sketch.js`**. 

Added missing semicolons and whitespace. Replaced some unnecessary for loops with array functions (mostly **`.reduce()`**).

I pulled the Style update before it being merged (No I didn't do magic, I just (correctly) guessed that it would be merged before I would have made a pr) so some commits come from there. (sorry for the mess 😬).

### CODE REVIEW

File: public/sketch.js [hunk 1]
```diff
@@ -1,116 +1,82 @@
...
+    if (firstRender)
+      doTheFirstRender(index, option, resultsDiv);
+
+    // Get the progress bar element
+    let progressBar = document.getElementById("progressBar_" + index);
+    // Set the width
+    progressBar.style.setProperty('width', width + '%');
+    // Set the text
+    progressBar.innerHTML = Math.round(count / totalVotes * 100) + '%';
+  }
```
Comment (shiffman) [inline public/sketch.js:60]: How would you feel about refactoring these to use p5.js functions: `select()`, `style()`, `html()`. I don't feel strongly but as of now other parts of the application are using p5 dom functions so might be good to be consistent?

File: public/sketch.js [hunk 2]
```diff
@@ -1,116 +1,82 @@
...
-  setInterval(countVotes, 500);
+const doTheFirstRender = (index, option, resultsDiv) => {
+  // Container div per option
+  let optionDiv = document.createElement('div');
+  optionDiv.classList.add('option');
 
-  let pollQ = createElement('p', poll.question)
-  pollQ.addClass("question")
+  // Add option text on a separate line, so it can get any length
+  let optionText = document.createElement('span');
```
Comment (shiffman) [inline public/sketch.js:70]: same question here!

File: public/sketch.js [hunk 3]
```diff
@@ -1,116 +1,82 @@
...
-    console.log(status);
-  } else {
-    console.log('no choice selected');
-  }
+  // Add option progress bar
+  let progressBar = document.createElement('div');
+
+  progressBar.id = 'progressBar_' + index;
+  progressBar.classList.add('progressBar');
+  resultsDiv.append(progressBar);
```
Comment (shiffman) [inline public/sketch.js:81]: I guess here too! Maybe we should just use native JS everywhere? No preference here!

**Comment by D-T-666** [comment]
yes good idea! done!

File: public/sketch.js (hunk 3 above)
Comment (D-T-666) [reply public/sketch.js]: Nah, p5 looks prettier and more readable, and also simpler.

**Comment by shiffman** [comment]
Great, thanks for the refactor!

**Pull request merged**